docs/api-docs/payments/payments-api-overview.md

```diff
@@ -264,6 +280,11 @@ Content-Type: application/json
 
 If the purchase was successful, the response returns a status of success. The order is then automatically moved to an Awaiting Fulfillment status. If you get a different response, see [Error codes](#error-codes) for troubleshooting.
 
+In the case of store credit and gift certificates:
+* If store credit and/or gift certificate covers the entire order amount, the order will be moved to an Awaiting Fulfillment status.
+* The order will stay in Pending status until it is fully paid. You can make the remaining order payment using other payment methods (Credit card, stored card, or store PayPal account) in the next payment request.
+
+For more information, see [Store Credit/Gift Certificate usage in S2S APIs](https://bc-eng-docs.herokuapp.com/domains/payments/technical-initiatives/s2s-store-credit-gift-certificate-usage.html).
```

Comment:
Could you please remove the docs link. It is an internal doc, had shared only for your reference.